src/main/cpp/blaze_util_windows.cc

```diff
@@ -415,6 +420,15 @@ string GetHomeDir() {
     ::CoTaskMemFree(wpath);
     return result;
   }
+
+  // On Windows 2016 Server, Datacenter edition, Nano server, FOLDERID_Profile
+  // us unknown but %USERPROFILE% is set.
```

Comment:
Nit: is?

Reply:
Ah, yes. I'll fix that when I import the PR>